lib/checks/aria/valid-focus-order-widget.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {HTMLElement} node
+ * @return {Boolean} Whether the element's role is appropriate for an element
+ *     in the focus order that isn't a scrollable region.
+ */
+function validFocusOrderWidget(node) {
```

Comment:
You don't need a separate function for this. The content of this file is wrapped in a function that will execute each iteration. This results in this function getting redeclared over and over.  You can just use the function body as the check code.

Reply:
Done.